onnxruntime/core/optimizer/propagate_cast_ops.cc

```diff
@@ -1094,34 +1129,34 @@ static bool PropagateFP16CastsFromOutputsToInputs(Graph& graph, Node* node,
           }
         }
       }
-      if (graph.IsOutput(output) && non_cast_consumers_map.find(output) == non_cast_consumers_map.end()) {
-        non_cast_consumers_map[output] = std::vector<Node*>();
+      if (graph.IsOutput(output) && !Contains(non_cast_consumers_map, output)) {
+        non_cast_consumers_map[output];
```

Comment:
>non_cast_consumers_map [](http://example.com/codeflow?start=8&length=22)

looks a bit odd - would an explicit call to emplace be clearer?

Reply:
Okay, will remove !Contains() here